<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajoutDepDossier</class>
 <widget class="QDialog" name="ajoutDepDossier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enregistrer Dépense/Dossier</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Res.qrc">
    <normaloff>:/icones/Icones/gnome_mime_application_qif.png</normaloff>:/icones/Icones/gnome_mime_application_qif.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Client</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Repertoire</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Date Reception</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Arrivé Bateau</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="clientComboBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="repEdit">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDateEdit" name="recepdateEdit">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QDateEdit" name="arrdateEdit">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Liste des dépenses du dossier</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTableView" name="contTableView">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btModifier">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Modifier</string>
              </property>
              <property name="icon">
               <iconset resource="../Res.qrc">
                <normaloff>:/icones/Icones/05-Edit-icon.png</normaloff>:/icones/Icones/05-Edit-icon.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btSupprimer">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Supprimer</string>
              </property>
              <property name="icon">
               <iconset resource="../Res.qrc">
                <normaloff>:/icones/Icones/14-Delete-icon.png</normaloff>:/icones/Icones/14-Delete-icon.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Ajouter dépense</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Agent</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Bon pour</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Date</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="agentEdit"/>
              </item>
              <item>
               <widget class="QPushButton" name="btAgent">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../Res.qrc">
                  <normaloff>:/icones/Icones/add.ico</normaloff>:/icones/Icones/add.ico</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLineEdit" name="bonEdit"/>
              </item>
              <item>
               <widget class="QPushButton" name="btDep">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../Res.qrc">
                  <normaloff>:/icones/Icones/add.ico</normaloff>:/icones/Icones/add.ico</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="1">
             <widget class="QDateEdit" name="dateEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Heure</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Montant</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QTimeEdit" name="timeEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="montantEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btAjouter">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Ajouter</string>
              </property>
              <property name="icon">
               <iconset resource="../Res.qrc">
                <normaloff>:/icones/Icones/add.ico</normaloff>:/icones/Icones/add.ico</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btValide">
              <property name="text">
               <string>Valider</string>
              </property>
              <property name="icon">
               <iconset resource="../Res.qrc">
                <normaloff>:/icones/Icones/15-Tick-icon.png</normaloff>:/icones/Icones/15-Tick-icon.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btEffacer">
              <property name="text">
               <string>Annuler</string>
              </property>
              <property name="icon">
               <iconset resource="../Res.qrc">
                <normaloff>:/icones/Icones/19-Left-Arrow-icon.png</normaloff>:/icones/Icones/19-Left-Arrow-icon.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btVoir">
       <property name="text">
        <string>Voir</string>
       </property>
       <property name="icon">
        <iconset resource="../Res.qrc">
         <normaloff>:/icones/Icones/10-Search-icon.png</normaloff>:/icones/Icones/10-Search-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btFermer">
       <property name="text">
        <string>Fermer</string>
       </property>
       <property name="icon">
        <iconset resource="../Res.qrc">
         <normaloff>:/icones/Icones/gnome_application_exit.png</normaloff>:/icones/Icones/gnome_application_exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>clientComboBox</tabstop>
  <tabstop>repEdit</tabstop>
  <tabstop>recepdateEdit</tabstop>
  <tabstop>arrdateEdit</tabstop>
  <tabstop>contTableView</tabstop>
  <tabstop>btModifier</tabstop>
  <tabstop>btSupprimer</tabstop>
  <tabstop>btAjouter</tabstop>
  <tabstop>btValide</tabstop>
  <tabstop>btEffacer</tabstop>
  <tabstop>btFermer</tabstop>
 </tabstops>
 <resources>
  <include location="../Res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btFermer</sender>
   <signal>clicked()</signal>
   <receiver>ajoutDepDossier</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>813</x>
     <y>545</y>
    </hint>
    <hint type="destinationlabel">
     <x>604</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btEffacer</sender>
   <signal>clicked()</signal>
   <receiver>bonEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>774</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>715</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btEffacer</sender>
   <signal>clicked()</signal>
   <receiver>montantEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>761</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>751</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btEffacer</sender>
   <signal>clicked()</signal>
   <receiver>btAgent</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>780</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>787</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btEffacer</sender>
   <signal>clicked()</signal>
   <receiver>agentEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>772</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>729</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
